modal/README.md

```diff
@@ -0,0 +1,60 @@
+# Modal
+
+## Overview
+
+[Modal][1] lets you run generative AI models, large-scale batch jobs, job
+queues, and much more. This integration collects metrics (CPU, memory
+or GPU use for example) and logs (stdout/stderr logging, audit logs) from your
+applications for you to observe directly in your Datadog account.
+
+## Setup
+
+### Installation
+
+To set up the Modal integration:
+
+1. Navigate to the [**Modal** tile][2] on the Datadog Integrations page and
+   click **Install Integration**.
+
+2. Click **Connect Accounts** to begin authorization of this integration. You
+   will be redirected to log into [Modal][1], and once logged in, you'll be
+   redirected to the Datadog authorization page.
+
+3. Click the **Authorize** button to complete setup.
+
+Logs and metrics from your Modal apps should now start appearing in Datadog.
+
+## Data Collected
+
+### Metrics
+
+Modal collects GPU, CPU, and memory utilization metrics for
+applications/functions that run on Modal.
+
+### Logs
+
+Modal collects audit logs and all application logs (all stdout/stderr logging).
```

Comment:
To confirm, the integration is sending logs to Datadog?

Reply:
Yes.